RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Open Food Facts - Flutter App - Release Guide
+
+Welcome to the release guide for the Open Food Facts Flutter App. Below are the steps to guide you through our 2 weeks release process.
+
+## 1. Pre-release Checklist
+
+Before you proceed with the release, ensure the following:
+
+- [ ] All the features and fixes planned for this release are merged into the `main` branch.
+- [ ] All merged features and fixes have corresponding unit and integration tests.
+- [ ] Update version number in the `pubspec.yaml` according to [semantic versioning](https://semver.org/). (Mergin Release Please should take care of that, and also provoke a F-Droid release up to a week later.)
+
+## 2. QA Testing
+
+- [ ] Refer to our QA Document to ensure that the app has been thoroughly tested against all release criteria. You can access the QA document [here](https://fake-url-to-QA-document.com).
+  
+## 3. Run Automated CI for Deployment
+
+1. [Run the CI GitHub Action to release to Internal and Testflight using this URL](https://github.com/openfoodfacts/smooth-app/actions/workflows/internal-release.yml).
+2. Trigger the automated CI process which will handle the build and deployment for both iOS and Android.
+3. [Go to the Play Console](https://play.google.com/console/u/0/developers/4712693179220384697/app/4972942602078310258/tracks/internal-testing), and add a changelog
+4. [Go to App Store Connect](https://appstoreconnect.apple.com/apps/588797948/testflight/ios), and select audiences to distribute to in the Testflight section, and add a Testflight changelog
+
+> **Note**: If there are any failures during the CI process, they must be addressed before proceeding.
+
+## 4. Verify Deployment
+
+- [ ] Once CI indicates successful internal deployment, download the app from both the Apple App Store and Google Play Store to ensure it's the latest version. Note that you need to register to Testflight or Play Console Internal to do that.
```

Comment:
We should add links for TestFlight (with an F) and G.Play

Reply:
fixed